mobile/src/home-view.js

```diff
@@ -170,8 +170,11 @@ class HomeView extends PureComponent {
           return name === item.replace('-Comment', '')
         })
         if (question) {
-          const answer = JSON.stringify(origResults[item])
-          let questionTitle = question.title ? question.title : question.name
+          const answer =
+            typeof origResults[item] === 'object'
```

Comment:
This seems to be doing the same localization as for title, so let's reuse the same functionality

Reply:
Should I make a new function or edit the current one to be more universal as the title function  is checking a variable of the main object called title

Reply:
Yeah, I'd say make the function something like:
```js
const getDefaultLocale = possiblyLocalized => typeof possiblyLocalized === 'object'
  ? possiblyLocalized.default
  : possiblyLocalized
```